### PR TITLE
Add missing endpoints on API page - Closes 1705

### LINKED ISSUE #1705: Add missing endpoints on API page
### Description
Some endpoints are missing on the RPC API page.

Maybe more are missing, this should be double-checked as well.

#### Affected branch / URL
`development`

#### Page location
`api/lisk-node-rpc.adoc`

#### Missing endpoints
- https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/engine/generator/endpoint.ts

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1705

### CODE REVIEW

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 1]
```diff
@@ -4056,139 +3304,2521 @@ curl --location --request POST 'http://localhost:7887/rpc' \
...
+[%collapsible]
+====
+.Example output
+[source,json]
+----
+{
+
+}
+----
+====
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:3646]: Is the response really an empty object?

**Review by Tschakki — APPROVED** [review]
LGTM 👍

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:3646]: Yes

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 2]
```diff
@@ -4056,139 +3304,2521 @@ curl --location --request POST 'http://localhost:7887/rpc' \
...
+
+
+==== `generator_setStatus`
+Used for setting the status a generator.
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]:
```suggestion
Used for setting the status of a generator.
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 3]
```diff
@@ -4056,139 +3304,2521 @@ curl --location --request POST 'http://localhost:7887/rpc' \
...
+--
+=====
+
+.Response
+[%collapsible]
+====
+.Example output
+[source,json]
+----
+{
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:4120]: it should return JSON response so we can update this.
Related issue https://github.com/LiskHQ/lisk-sdk/issues/8210

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 4]
```diff
@@ -4056,139 +3304,2521 @@ curl --location --request POST 'http://localhost:7887/rpc' \
...
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:4180]: i think its missing chainID, also genesis property is missing `block`

**Review by ishantiw — COMMENTED** [review]
Nicely done, left some minor comments

**Review by ishantiw — APPROVED** [review]
🎉

**Pull request merged**